src/openforms/js/components/admin/forms/ReactSelect.js

```diff
@@ -0,0 +1,65 @@
+import {getReactSelectStyles} from '@open-formulieren/formio-builder/esm/components/formio/select';
+import {useField} from 'formik';
+import PropTypes from 'prop-types';
+import ReactSelect from 'react-select';
+
+const initialStyles = getReactSelectStyles();
```

Comment:
I noticed that the size of the selectbox scales with the search input, while it isn't major, I guess it would be nicer if it stayed the same size
![Screenshot from 2024-08-06 12-31-18](https://github.com/user-attachments/assets/7e4f39a0-3675-4903-8fe1-6ae82dd00e6b)

Reply:
I can't seem to reproduce this - maybe browser CSS cache? I've tested on Chromium and Firefox.